services/resource-usage-tracker/tests/unit/test_resource_tracker_core.py

```diff
@@ -8,7 +8,7 @@
 import pytest
 import requests_mock
 from fastapi import FastAPI
-from simcore_service_resource_usage_tracker.resource_tracker_core import (
+from simcore_service_resource_usage_tracker.resource_tracker_cli_placeholder import (
```

Comment:
I dont understand why do you import the legacy devops PoC here :/

Reply:
The same as above

Reply:
same, see above, potentially comment